RobotCrawler/src/main/java/parser/ebookscom/EBooksParser.java

```diff
@@ -0,0 +1,23 @@
+package parser.ebookscom;
+
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import parser.Parser;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by ehsan on 25.08.16.
+ */
+public class EBooksParser implements Parser<Element> {
+
+    @Override
+    public List parse(Document document) {
+        // all results. in ebooks.com the result of a category are stored int the list,
```

Comment:
@LIttleAncientForestKami 
It's IntelliJ false, it kept generating those shitty comments "Created by .... on ...." in my case too. I turned this off